core-web/libs/template-builder/src/lib/components/template-builder/components/add-style-classes-dialog/add-style-classes-dialog.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="dialog__auto-complete-container">
+    <p class="auto-complete-container__title">Class</p>
+    <div class="p-fluid">
+        <p-autoComplete
+            [(ngModel)]="selectedClasses"
+            [suggestions]="filteredClasses"
+            [multiple]="true"
+            [completeOnFocus]="true"
+            [size]="446"
+            (completeMethod)="filterClasses($event)"
+            field="cssClass"
```

Comment:
BEM

Reply:
the field refers to what `property` of the `object` will be using to display in the `autocomplete chips`